Known issue: incompatibility with Jackson 2.17.0-rc1

With `jackson-databind` version `2.17.0-rc1`, the JSON encoding/decoding test for `MetadataBLOBHeader` [fails](https://github.com/Yubico/java-webauthn-server/actions/runs/8134017406/job/22226343713):

```
com.yubico.fido.metadata.JsonIoSpec > The class com.yubico.fido.metadata.MetadataBLOBHeader is identical after multiple serialization round-trips. FAILED
    org.scalatest.exceptions.GeneratorDrivenPropertyCheckFailedException at JsonIoSpec.scala:51
        Caused by: com.fasterxml.jackson.databind.JsonMappingException at JsonIoSpec.scala:52
            Caused by: java.lang.ClassCastException at JsonIoSpec.scala:52
```

The cause of the test failure is:
```
org.scalatest.exceptions.GeneratorDrivenPropertyCheckFailedException: JsonMappingException was thrown during property evaluation.
  Message: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap') (through reference chain: com.yubico.fido.metadata.MetadataBLOBHeader["x5c"])
  Occurred when passed generated values (
    arg0 = MetadataBLOBHeader(typ=Optional[JWT], alg=, x5u=Optional[http:], x5c=Optional[[]])
  )
```

The issue appears with java-webauthn-server versions `2.0.0` and `2.5.0`. The issue does not appear with Jackson version `2.16.1`.

<details><summary>Full stack trace</summary>
<pre>
org.scalatest.exceptions.GeneratorDrivenPropertyCheckFailedException: JsonMappingException was thrown during property evaluation.
  Message: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap') (through reference chain: com.yubico.fido.metadata.MetadataBLOBHeader["x5c"])
  Occurred when passed generated values (
    arg0 = MetadataBLOBHeader(typ=Optional[JWT], alg=, x5u=Optional[http:], x5c=Optional[[]])
  )
Init Seed: 4625492476167262463
	at app//org.scalatestplus.scalacheck.CheckerAsserting$$anon$2.indicateFailure(CheckerAsserting.scala:253)
	at app//org.scalatestplus.scalacheck.CheckerAsserting$$anon$2.indicateFailure(CheckerAsserting.scala:240)
	at app//org.scalatestplus.scalacheck.UnitCheckerAsserting$CheckerAssertingImpl.check(CheckerAsserting.scala:162)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.apply(ScalaCheckDrivenPropertyChecks.scala:579)
	at app//com.yubico.fido.metadata.JsonIoSpec.$anonfun$new$3(JsonIoSpec.scala:51)
	at app//org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
	at app//org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
	at app//org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
	at app//org.scalatest.Transformer.apply(Transformer.scala:22)
	at app//org.scalatest.Transformer.apply(Transformer.scala:20)
	at app//org.scalatest.funspec.AnyFunSpecLike$$anon$1.apply(AnyFunSpecLike.scala:517)
	at app//org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
	at app//org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
	at app//org.scalatest.funspec.AnyFunSpec.withFixture(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.invokeWithFixture$1(AnyFunSpecLike.scala:515)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$runTest$1(AnyFunSpecLike.scala:527)
	at app//org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTest(AnyFunSpecLike.scala:527)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTest$(AnyFunSpecLike.scala:509)
	at app//org.scalatest.funspec.AnyFunSpec.runTest(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$runTests$1(AnyFunSpecLike.scala:560)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:413)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:396)
	at app//org.scalatest.SuperEngine.runTestsImpl(Engine.scala:475)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTests(AnyFunSpecLike.scala:560)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTests$(AnyFunSpecLike.scala:559)
	at app//org.scalatest.funspec.AnyFunSpec.runTests(AnyFunSpec.scala:1632)
	at app//org.scalatest.Suite.run(Suite.scala:1114)
	at app//org.scalatest.Suite.run$(Suite.scala:1096)
	at app//org.scalatest.funspec.AnyFunSpec.org$scalatest$funspec$AnyFunSpecLike$$super$run(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$run$1(AnyFunSpecLike.scala:564)
	at app//org.scalatest.SuperEngine.runImpl(Engine.scala:535)
	at app//org.scalatest.funspec.AnyFunSpecLike.run(AnyFunSpecLike.scala:564)
	at app//org.scalatest.funspec.AnyFunSpecLike.run$(AnyFunSpecLike.scala:563)
	at app//org.scalatest.funspec.AnyFunSpec.run(AnyFunSpec.scala:1632)
	at app//org.scalatestplus.junit.JUnitRunner.run(JUnitRunner.scala:99)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:108)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:40)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:60)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:52)
	at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.10/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.10/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:176)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper canorg.scalatest.exceptions.GeneratorDrivenPropertyCheckFailedException: JsonMappingException was thrown during property evaluation.
  Message: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap') (through reference chain: com.yubico.fido.metadata.MetadataBLOBHeader["x5c"])
  Occurred when passed generated values (
    arg0 = MetadataBLOBHeader(typ=Optional[JWT], alg=, x5u=Optional[http:], x5c=Optional[[]])
  )
Init Seed: 4625492476167262463
	at app//org.scalatestplus.scalacheck.CheckerAsserting$$anon$2.indicateFailure(CheckerAsserting.scala:253)
	at app//org.scalatestplus.scalacheck.CheckerAsserting$$anon$2.indicateFailure(CheckerAsserting.scala:240)
	at app//org.scalatestplus.scalacheck.UnitCheckerAsserting$CheckerAssertingImpl.check(CheckerAsserting.scala:162)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.apply(ScalaCheckDrivenPropertyChecks.scala:579)
	at app//com.yubico.fido.metadata.JsonIoSpec.$anonfun$new$3(JsonIoSpec.scala:51)
	at app//org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
	at app//org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
	at app//org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
	at app//org.scalatest.Transformer.apply(Transformer.scala:22)
	at app//org.scalatest.Transformer.apply(Transformer.scala:20)
	at app//org.scalatest.funspec.AnyFunSpecLike$$anon$1.apply(AnyFunSpecLike.scala:517)
	at app//org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
	at app//org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
	at app//org.scalatest.funspec.AnyFunSpec.withFixture(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.invokeWithFixture$1(AnyFunSpecLike.scala:515)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$runTest$1(AnyFunSpecLike.scala:527)
	at app//org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTest(AnyFunSpecLike.scala:527)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTest$(AnyFunSpecLike.scala:509)
	at app//org.scalatest.funspec.AnyFunSpec.runTest(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$runTests$1(AnyFunSpecLike.scala:560)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:413)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
	at app//org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
	at app//scala.collection.immutable.List.foreach(List.scala:333)
	at app//org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
	at app//org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:396)
	at app//org.scalatest.SuperEngine.runTestsImpl(Engine.scala:475)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTests(AnyFunSpecLike.scala:560)
	at app//org.scalatest.funspec.AnyFunSpecLike.runTests$(AnyFunSpecLike.scala:559)
	at app//org.scalatest.funspec.AnyFunSpec.runTests(AnyFunSpec.scala:1632)
	at app//org.scalatest.Suite.run(Suite.scala:1114)
	at app//org.scalatest.Suite.run$(Suite.scala:1096)
	at app//org.scalatest.funspec.AnyFunSpec.org$scalatest$funspec$AnyFunSpecLike$$super$run(AnyFunSpec.scala:1632)
	at app//org.scalatest.funspec.AnyFunSpecLike.$anonfun$run$1(AnyFunSpecLike.scala:564)
	at app//org.scalatest.SuperEngine.runImpl(Engine.scala:535)
	at app//org.scalatest.funspec.AnyFunSpecLike.run(AnyFunSpecLike.scala:564)
	at app//org.scalatest.funspec.AnyFunSpecLike.run$(AnyFunSpecLike.scala:563)
	at app//org.scalatest.funspec.AnyFunSpec.run(AnyFunSpec.scala:1632)
	at app//org.scalatestplus.junit.JUnitRunner.run(JUnitRunner.scala:99)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:108)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:40)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:60)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:52)
	at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.10/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.10/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:176)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap') (through reference chain: com.yubico.fido.metadata.MetadataBLOBHeader["x5c"])
	at app//com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:402)
	at app//com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:361)
	at app//com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:323)
	at app//com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:778)
	at app//com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:183)
	at app//com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:502)
	at app//com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:341)
	at app//com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4799)
	at app//com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:4040)
	at app//com.yubico.fido.metadata.JsonIoSpec.$anonfun$new$4(JsonIoSpec.scala:52)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.liftedTree1$1(ScalaCheckDrivenPropertyChecks.scala:559)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.$anonfun$apply$1(ScalaCheckDrivenPropertyChecks.scala:558)
	at app//scala.Function1.$anonfun$andThen$1(Function1.scala:85)
	at app//org.scalacheck.Prop$.$anonfun$forAllShrink$2(Prop.scala:768)
	at app//org.scalacheck.Prop$.secure(Prop.scala:478)
	at app//org.scalacheck.Prop$.result$1(Prop.scala:768)
	at app//org.scalacheck.Prop$.$anonfun$forAllShrink$1(Prop.scala:807)
	at app//org.scalacheck.Prop$.$anonfun$apply$1(Prop.scala:308)
	at app//org.scalacheck.PropFromFun.apply(Prop.scala:21)
	at app//org.scalacheck.Test$.workerFun$1(Test.scala:434)
	at app//org.scalacheck.Test$.$anonfun$check$2(Test.scala:466)
	at app//org.scalacheck.Test$.$anonfun$check$2$adapted(Test.scala:466)
	at app//org.scalacheck.Platform$.runWorkers(Platform.scala:40)
	at app//org.scalacheck.Test$.check(Test.scala:466)
	at app//org.scalatestplus.scalacheck.UnitCheckerAsserting$CheckerAssertingImpl.check(CheckerAsserting.scala:92)
	... 66 more
Caused by: java.lang.ClassCastException: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap')
	at com.yubico.fido.metadata.CertToBase64Converter.convert(CertToBase64Converter.java:10)
	at com.fasterxml.jackson.databind.ser.std.StdDelegatingSerializer.convertValue(StdDelegatingSerializer.java:264)
	at com.fasterxml.jackson.databind.ser.std.StdDelegatingSerializer.serialize(StdDelegatingSerializer.java:156)
	at com.fasterxml.jackson.databind.ser.std.ReferenceTypeSerializer.serialize(ReferenceTypeSerializer.java:389)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:732)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
	... 87 morenot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap') (through reference chain: com.yubico.fido.metadata.MetadataBLOBHeader["x5c"])
	at app//com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:402)
	at app//com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:361)
	at app//com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:323)
	at app//com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:778)
	at app//com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:183)
	at app//com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:502)
	at app//com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:341)
	at app//com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4799)
	at app//com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:4040)
	at app//com.yubico.fido.metadata.JsonIoSpec.$anonfun$new$4(JsonIoSpec.scala:52)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.liftedTree1$1(ScalaCheckDrivenPropertyChecks.scala:559)
	at app//org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks$ConfiguredPropertyCheck.$anonfun$apply$1(ScalaCheckDrivenPropertyChecks.scala:558)
	at app//scala.Function1.$anonfun$andThen$1(Function1.scala:85)
	at app//org.scalacheck.Prop$.$anonfun$forAllShrink$2(Prop.scala:768)
	at app//org.scalacheck.Prop$.secure(Prop.scala:478)
	at app//org.scalacheck.Prop$.result$1(Prop.scala:768)
	at app//org.scalacheck.Prop$.$anonfun$forAllShrink$1(Prop.scala:807)
	at app//org.scalacheck.Prop$.$anonfun$apply$1(Prop.scala:308)
	at app//org.scalacheck.PropFromFun.apply(Prop.scala:21)
	at app//org.scalacheck.Test$.workerFun$1(Test.scala:434)
	at app//org.scalacheck.Test$.$anonfun$check$2(Test.scala:466)
	at app//org.scalacheck.Test$.$anonfun$check$2$adapted(Test.scala:466)
	at app//org.scalacheck.Platform$.runWorkers(Platform.scala:40)
	at app//org.scalacheck.Test$.check(Test.scala:466)
	at app//org.scalatestplus.scalacheck.UnitCheckerAsserting$CheckerAssertingImpl.check(CheckerAsserting.scala:92)
	... 66 more
Caused by: java.lang.ClassCastException: class scala.collection.convert.JavaCollectionWrappers$SeqWrapper cannot be cast to class java.security.cert.X509Certificate (scala.collection.convert.JavaCollectionWrappers$SeqWrapper is in unnamed module of loader 'app'; java.security.cert.X509Certificate is in module java.base of loader 'bootstrap')
	at com.yubico.fido.metadata.CertToBase64Converter.convert(CertToBase64Converter.java:10)
	at com.fasterxml.jackson.databind.ser.std.StdDelegatingSerializer.convertValue(StdDelegatingSerializer.java:264)
	at com.fasterxml.jackson.databind.ser.std.StdDelegatingSerializer.serialize(StdDelegatingSerializer.java:156)
	at com.fasterxml.jackson.databind.ser.std.ReferenceTypeSerializer.serialize(ReferenceTypeSerializer.java:389)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:732)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
	... 87 more
</pre>
</details>

--- a/NEWS
+++ b/NEWS
@@ -3,8 +3,7 @@
 Changes:
 
 * Dropped dependency on COSE-Java.
-* Excluded Jackson version 2.17.0-rc1 from dependency resolution due to an
-  incompatible regression.
+* Fixed incompatibility with Jackson version 2.17.0-rc1.
 
 
 == Version 2.5.0 ==

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -19,10 +19,7 @@ dependencyResolutionManagement {
             library("httpclient5", "org.apache.httpcomponents.client5:httpclient5:[5.0.0,6)")
             library("slf4j", "org.slf4j:slf4j-api:[1.7.25,3)")
 
-            val jacksonVer = version("jackson") {
-                require("[2.13.2.1,3)")
-                reject("2.17.0-rc1") // Regression: https://github.com/FasterXML/jackson-databind/issues/4413
-            }
+            val jacksonVer = version("jackson", "[2.13.2.1,3)")
             library("jackson-bom", "com.fasterxml.jackson", "jackson-bom").versionRef(jacksonVer)
             library("jackson-databind", "com.fasterxml.jackson.core", "jackson-databind").versionRef(jacksonVer)
             library("jackson-dataformat-cbor", "com.fasterxml.jackson.dataformat", "jackson-dataformat-cbor").versionRef(jacksonVer)

--- a/webauthn-server-attestation/src/main/java/com/yubico/fido/metadata/MetadataBLOBHeader.java
+++ b/webauthn-server-attestation/src/main/java/com/yubico/fido/metadata/MetadataBLOBHeader.java
@@ -1,5 +1,6 @@
 package com.yubico.fido.metadata;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.net.URL;
@@ -86,6 +87,9 @@ public class MetadataBLOBHeader {
    * @see <a href="https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.6">RFC 7515 §4.1.6.
    *     "x5c" (X.509 Certificate Chain) Header Parameter</a>
    */
+  // @JsonIgnore needed because of:
+  // https://github.com/FasterXML/jackson-databind/issues/4413#issuecomment-1977989776
+  @JsonIgnore
   public Optional<List<X509Certificate>> getX5c() {
     return Optional.ofNullable(x5c);
   }

